.gitignore

```diff
@@ -15,3 +15,6 @@ Thumbs.db
 # Files that might appear on external disks
 .Spotlight-V100
 .Trashes
+
+# Generated files
+punycode.es6.js
```

Comment:
Let’s move this to the top of `.gitignore`. Project-specific files deserve to go first!